ui/src/App.tsx

```diff
@@ -21,8 +21,11 @@ import {
 import { Link, Outlet } from "react-router-dom";
 import "./App.scss";
 import { ExternalLinks, Links } from "./router/links.models";
+import axios from "axios";
 
 function App() {
+  axios.defaults.baseURL = import.meta.env.VITE_WANAKU_ROUTER_BASE_URL || 'http://localhost:8080';
```

Comment:
I think we could use the env variable directly `VITE_API_URL` because it's populated by vite itself here: https://github.com/wanaku-ai/wanaku/blob/ce17e4749829df0aedcd24c23e9f81f037e1dadb/ui/vite.config.ts#L12

When building it with quarkus, quinoa sets the information here: https://github.com/wanaku-ai/wanaku/blob/ce17e4749829df0aedcd24c23e9f81f037e1dadb/routers/wanaku-router/src/main/resources/application.properties#L37 so it's relative to the project URL.

Reply:
I am using VITE_API_URL, but it still seems a bit hacky, the `replaceAll` in particular